components/atom/button/src/index.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const CLASS = 'sui-AtomButton'
+const TYPES = ['primary', 'accent', 'secondary', 'tertiary']
+const SIZES = ['small', 'large']
```

Comment:
what about exporting these props so they can be used to feed the component without magic strings?

Reply:
Thinking about remaining as similar as HTML spec I thought it wasn't necessary.
We agreed not to force the use of constants for easier implementation. But maybe we still want to expose them for those who prefer to use them ?

Reply:
I guess it's up to you